### PR TITLE
changed tam.json to ta.json and minor changes

### PR DESCRIPTION
**Pull request by AadharshAadhithya**

### CODE REVIEW

File: en.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
 			"option-spellcheck": "Spell check",
 			"option-spellcheck-description": "Turns on the spell checker. Currently only works for English (US).",
 			"spellcheck-dict": "Spellcheck dictionary",
-			"spellcheck-dict-empty": "The spellcheck dictionary is empty.",
+			"spellcheck-dict-emptåy": "The spellcheck dictionary is empty.",
```
Comment (ericaxu) [inline en.json:12]: Please do not translate `en.json`, thanks!

**Review by ericaxu — CHANGES_REQUESTED** [review]

File: en.json (hunk 1 above)
Comment (AadharshAadhithya) [reply en.json:12]: Ok , that was a typo thou... thanks anyways.. :)

Comment (ericaxu) [reply en.json:12]: Ah okay. Once you have the time to change it back I can merge it, thanks!

**Comment by ericaxu** [comment]
Could you restore `en.json` can I can merge this? Thanks.

**Pull request closed without merging**